src/components/CollaboratorModal/components/AcknowledgementSubmodal.tsx

```diff
@@ -72,7 +72,11 @@ export const AcknowledgementSubmodalContent = ({
       <Checkbox {...register("isAcknowledged")}>
         <Text color="text.body" fontSize="16px">
           <Text as="span">I agree to Isomer&lsquo;s</Text>{" "}
-          <Link href={TERMS_OF_USE_LINK} target="_blank">
+          <Link
+            href={TERMS_OF_USE_LINK}
+            target="_blank"
+            rel="noopener noreferrer"
```

Comment:
can consider using `isExternal` - it does this and is clearer but not a major issue

Reply:
I tried using `isExternal` but unfortunately it adds an additional icon at the back to indicate that it is an external link, which is not what we want to show here.